src/commands/init.js

```diff
@@ -168,46 +167,6 @@ const logExistingRepoSetupAndExit = ({ log, exit, siteName, repoUrl }) => {
   exit()
 }
 
-const configureGitHub = async ({ context, siteInfo, repo }) => {
-  try {
-    await configGithub(context, siteInfo, repo)
-  } catch (error) {
-    context.warn(`GitHub error: ${error.status}`)
-    if (error.status === 404) {
-      context.error(
-        `Does the repository ${repo.repo_path} exist and do you have the correct permissions to set up deploy keys?`,
-      )
-    } else {
-      throw error
-    }
-  }
-}
-
-const configureProvider = async ({ context, siteInfo, repo }) => {
-  switch (repo.provider) {
```

Comment:
This was duplicate with https://github.com/netlify/cli/blob/bd59e01b8f4214035e6f69bd85182008bb810f07/src/commands/sites/create.js#L141